romancal/associations/mk_skycell_list.py

```diff
@@ -1,59 +1,55 @@
-"""Create a list of patches for a level 2 file or files based on skycells"""
+"""Create a list of skycells for a level 2 file or files based on skycells"""
 
 import argparse
 import logging
 import os.path
 import sys
 
-import numpy as np
 import roman_datamodels as rdm
 
-import romancal.patch_match.patch_match as pm
+import romancal.skycell.match as sm
 
-__all__ = ["mk_patchlist"]
+__all__ = ["mk_skycell_list"]
 
 # Configure logging
 logger = logging.getLogger(__name__)
 logger.addHandler(logging.NullHandler())
 logger.setLevel("INFO")
 
 
-def mk_patchlist(output_dir, filelist):
+def mk_skycell_list(output_dir, filelist):
     """
     Create a list of skycell id's based on a list of level 2 files.
 
     This function processes a list of calibrated detector files,
-    identifies matching patches on the sky and creates
-    file with that list of sky patches the level 2 file touches.
+    identifies matching skycells on the sky and creates
+    file with that list of skycells the level 2 file touches.
 
     Parameters
     ----------
     filelist : list or str
         List of file names to be processed.
     output_dir : str
-        Directory for the output patch file.
+        Directory for the output skycell file.
     """
 
     for file_name in filelist:
         input_dir, input_file = os.path.split(file_name)
         cal_file = rdm.open(file_name)
-        file_patch_list = pm.find_patch_matches(cal_file.meta.wcs)
-        logger.info(f"Patch List:{file_name}, {file_patch_list[0]}")
+        intersecting_skycells = sm.find_skycell_matches(cal_file.meta.wcs)
+        logger.info(f"Skycell List: {file_name}, {intersecting_skycells}")
         output_file_name = os.path.basename(input_file).split(".")[0]
         if not output_dir:
             output_file_name = os.path.join(input_dir, output_file_name)
         else:
             output_file_name = os.path.join(output_dir, output_file_name)
 
         with open(output_file_name + ".match", "w") as outfile:
-            out_string = (
-                file_name + " " + np.array2string(file_patch_list[0], separator=",")
-            )
-            outfile.write(out_string)
```

Comment:
It looks like mk_patchlist / mk_skycell_asn_from_skycell_list rely on the precise formatting this np.array2string(...) gives.  With the changes this string gets more spaces, which are then parsed incorrectly around line 43 of mk_skycell_asn_from_skycell_list.  This then leads to the smaller number of output skycells.  One  could imagine switching to some less bespoke format, but probably let's just retain the original behavior for this PR?